### PR TITLE
Added extra fields of input in register business

### PR DESCRIPTION
**Pull request by ritwizsinha**
Changes: Added the fields of opening,closing time,established date,email,website address to the register business page

Surge Deployment Link: https://pr-124-sahaayak.surge.sh

Screenshots for the change: [Add here screenshots showing the changes.]
Screenshot from 2019-05-27 16-09-45

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@ritwizsinha Surge link ?

**Comment by ritwizsinha** [comment]
Added it

File: src/components/VendorSalesPage/VendorSalesPage.js [hunk 1]
```diff
@@ -162,6 +173,49 @@ class VendorSalesPage extends Component {
...
+                message: 'The input is not valid E-mail!',
+              },
+              {
+                required: false
+              },
+            ],
+          })(<Input />)}
+          </Form.Item>
+          <Form.Item
+              label="Register your website"
```
Comment (harsh-jindal) [inline src/components/VendorSalesPage/VendorSalesPage.js]: Change this to `Link your website`.
Also change habitual residence to shop address.

File: src/components/VendorSalesPage/VendorSalesPage.js [hunk 2]
```diff
@@ -162,6 +173,49 @@ class VendorSalesPage extends Component {
...
+              {getFieldDecorator('website', {
+                rules: [{ required: false, whitespace: true }],
+              })(
+                <Input placeholder="Enter Website" />
+              )}
+          </Form.Item>
+          <Form.Item label="Opening Time">
+          {getFieldDecorator('openingTime',{
+            rules: [{required:true}],
+          })( <TimePicker use12Hours format="h:mm a"  />)}
```
Comment (harsh-jindal) [inline src/components/VendorSalesPage/VendorSalesPage.js]: Currently the user can set closing time before opening time. Correct that.
Also the antd timepicker isn't that great. Try finding something which is more user friendly. 
You may use different library for it. Before implementing this please share the link of the library.

**Review by harsh-jindal — CHANGES_REQUESTED** [review]

**Comment by harsh-jindal** [comment]
@ritwizsinha Make the above changes also are these field implemented in the server. If not first finish the implementation of all basic fields in the server then continue on this.

File: src/components/VendorSalesPage/VendorSalesPage.js (hunk 1 above)
Comment (ritwizsinha) [reply src/components/VendorSalesPage/VendorSalesPage.js]: Done

**Pull request closed without merging**